src/test/java/com/baidu/hugegraph/api/VertexApiTest.java

```diff
@@ -210,22 +214,94 @@ public void testBatchCreate() {
     }
 
     @Test
-    public void testBatchUpdateProperties() {
-        Map<String, Object> strategies = ImmutableMap.of("set", "union",
-                                                         "fullDate", "smaller");
-        // Init old vertices
-        graph().addVertices(this.createNVertexBatch("testV", 5, "old"));
-        List<Vertex> vertices = this.createNVertexBatch("testV", 5, "new");
-
-        // TODO: List from server is unordered, need consider better way to test
-        graph().updateVertices(vertices, strategies, true).forEach(vertex -> {
-            Object object = vertex.properties().get("set");
-            Assert.assertTrue(object instanceof List);
-            // Vertex's property has pid now, diff with edge
-            Map maps = (Map) (((List) object).get(0));
-            Object sets = maps.get("value");
-            Assert.assertTrue(sets instanceof List);
-            Assert.assertTrue(maps.size() == 2 && ((List) sets).size() == 2);
+    public void testBatchUpdateStrategySum() {
+        BatchVertexRequest req = batchVertexRequest("price", 1, -1,
+                                                    UpdateStrategy.SUM);
+
+        this.graph().updateVertices(req).forEach(vertex -> {
+            Object price = vertex.properties().get("price");
+            Assert.assertTrue(price instanceof Number);
+            Assert.assertTrue((int) price == 0);
+            System.out.println(vertex);
+        });
+    }
+
+    @Test
+    public void testBatchUpdateStrategyBigger() {
+        BatchVertexRequest req = batchVertexRequest("price", 1, -1,
+                                                    UpdateStrategy.BIGGER);
+
+        this.graph().updateVertices(req).forEach(vertex -> {
+            Object price = vertex.properties().get("price");
+            Assert.assertTrue(price instanceof Number);
+            Assert.assertTrue((int) price > 0);
+            System.out.println(vertex);
+        });
+    }
+
+    @Test
+    public void testBatchUpdateStrategySmaller() {
+        BatchVertexRequest req = batchVertexRequest("fullDate", -1, 1,
```

Comment:
Don't use fullDate(Date property), it should actually be deserialized to date, now being deserialized to number is a bug